targets/targets.json

```diff
@@ -2109,7 +2113,8 @@
             }
         },
         "overrides": {
-            "lpticker_delay_ticks": 0
+            "lpticker_delay_ticks": 0,
+            "flash-non-uniform-sectors": true
```

Comment:
I think STM32F2 and F4 are also concerned.

Reply:
@jeromecoutant @LMESTM That's right. The only thing is adding F2 and F4 will increase the number of affected targets in this PR, but if you are happy with that, I'll do.